core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -1225,7 +1225,7 @@ private ConnectorOverview filterByEnvironment(
       List<KwKafkaConnector> connectors, String envId, int tenantId) {
     ConnectorOverview overview = new ConnectorOverview();
     String orderOfEnvs = commonUtilsService.getEnvProperty(tenantId, ORDER_OF_KAFKA_CONNECT_ENVS);
-    List<String> orderOfEnvsArrayList = KlawResourceUtils.getOrderedEnvsList(orderOfEnvs);
+    Set<String> orderOfEnvsArrayList = KlawResourceUtils.getOrderedEnvsSet(orderOfEnvs);
```

Comment:
We should probably change this from -> to
orderOfEnvsArrayList -> orderOfEnvsArraySet

Reply:
done